session/session.go

```diff
@@ -4283,3 +4278,24 @@ func RemoveLockDDLJobs(s Session, job2ver map[int64]int64, job2ids map[int64]str
 		return true
 	})
 }
+
+// GetDBNames gets the sql layer database names from the session.
+func GetDBNames(se *session) []string {
```

Comment:
Nearly the same function with https://github.com/pingcap/tidb/pull/42026/files#diff-412615e1c1d4c6ae11a7885a11c879778bf401e0967c9c13c4b9dfd1f13ed29aR2561